Jenkinsfile

```diff
@@ -119,6 +143,9 @@ try {
     stage("Sanity Check") {
       timeout(time: max_time, unit: 'MINUTES') {
         node('mxnetlinux') {
+          if (env.BRANCH_NAME != "master") {
```

Comment:
We can also change this condition to run this only on PR builds. Let me know if that is preferable.

Reply:
I don't know what is the right answer here, please check with others on slack. We can run this now on the PR builds and then remove this condition later.